spec/support/schema.rb

```diff
@@ -33,6 +33,7 @@ class Person < ActiveRecord::Base
   belongs_to :parent, :class_name => 'Person', :foreign_key => :parent_id
   has_many   :children, :class_name => 'Person', :foreign_key => :parent_id
   has_many   :articles
+  has_many   :published_articles, :class_name => 'Article', :conditions => {published: true}
```

Comment:
Line is too long. [92/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.